### DISCUSSION TITLE
how to run on raspberry pi - arm64?

### DISCUSSION THREAD

**Discussion by user9931** [IgnisDa/ryot#539] [Q&A] [answered]
maybe i'm doing something wrong...

trying docker-compose in documentation and downloading postgres, postgres of course starts, then ryot and... 
ryot             | exec /usr/local/bin/docker-entrypoint.sh: exec format error

so thinks that arm64 are not supported, right? first time see images on github, but https://github.com/ignisda/ryot/pkgs/container/ryot/162586668?tag=latest shows there's amr64 and arm64. Trying to specify arm64 - ghcr.io/ignisda/ryot:v4.0.5@sha256:58612bddc810bf8f7d32b0329dd4431dabf1baf3717965e4ed1f61614e68af24  in docker-compose, but it gives same exec format error...

**Comment by IgnisDa** [comment]
@vnghia Any idea why this might be happening?

**Reply by vnghia** [reply, marked as answer]
this should fix the error: https://github.com/IgnisDa/ryot/pull/541

**Comment by vnghia** [comment]
@user9931 can you try `ryot:v4.0.8-beta1`

**Reply by user9931** [reply]
yeap, now it started! great!

**Reply by IgnisDa** [reply]
Thanks @vnghia!
